README.md

```diff
@@ -59,18 +59,14 @@ Now, you will be able to open the solution, and simply build it in Visual Studio
 ## Staying Up To Date
 
 Pull the latest changes from Github:
```

Comment:
Don't forget to remove this.

Reply:
Ah yes

Reply:
> Don't forget to remove this.
> 
> ```
> Pull the latest changes from Github:
> ```

I think it shuld be:
```markdown
Pull the latest changes from Github and regenerate project files:
```
To not confuse people.

Reply:
Sure that could be done

Reply:
Wait I accidentally removed somethingg